services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/android/attestation/ProdSrsRateLimitVerificationStrategyTest.java

```diff
@@ -29,7 +30,7 @@ final class ProdSrsRateLimitVerificationStrategyTest {
 
   @Test
   void testInvalidSrsRateLimit() {
-    final AndroidId androidId = new AndroidId();
+    final AndroidId androidId = new AndroidId(UUID.randomUUID().toString());
```

Comment:
can we not just use the non-args constructor and then use setId?
```suggestion
    final AndroidId androidId = new AndroidId();
    androidId.setId(UUID.randomUUID().toString());
```

Reply:
I took this over from the way OneTimePassword works. I can't say 100% if this affects anything. But all tests still work.